Dockerfile

```diff
@@ -101,6 +101,7 @@ RUN --mount=type=bind,target=. \
 FROM build-base AS test
 ARG CGO_ENABLED=0
 ARG BUILD_TAGS
+ENV COMPOSE_MENU=FALSE
```

Comment:
Is this necessary for building?

Reply:
yes, because by default it should be true, in order not to break tests this is easier to set the COMPOSE_MENU false in all containers